tests/e2e/backup_restore_suite_test.go

```diff
@@ -16,6 +16,31 @@ import (
 
 type VerificationFunction func(client.Client, string) error
 
+func mongoready(preBackupState bool, backupRestoreType BackupRestoreType) VerificationFunction {
+	return VerificationFunction(func(ocClient client.Client, namespace string) error {
+		Eventually(IsDCReady(ocClient, namespace, "todolist"), timeoutMultiplier*time.Minute*10, time.Second*10).Should(BeTrue())
+		err := VerifyBackupRestoreData(artifact_dir, namespace, "todolist-route", "todolist", preBackupState, backupRestoreType) // TODO: VERIFY PARKS APP DATA
+		return err
+	})
+}
+func mysqlReady(preBackupState bool, backupRestoreType BackupRestoreType) VerificationFunction {
+	return VerificationFunction(func(ocClient client.Client, namespace string) error {
+		fmt.Printf("checking for the NAMESPACE: %s\n ", namespace)
+		// This test confirms that SCC restore logic in our plugin is working
+		//Eventually(IsDCReady(ocClient, "mssql-persistent", "mysql"), timeoutMultiplier*time.Minute*10, time.Second*10).Should(BeTrue())
+		Eventually(IsDeploymentReady(ocClient, namespace, "mysql"), timeoutMultiplier*time.Minute*10, time.Second*10).Should(BeTrue())
+		exists, err := DoesSCCExist(ocClient, "mysql-persistent-scc")
+		if err != nil {
+			return err
+		}
+		if !exists {
+			return errors.New("did not find MYSQL scc")
```

Comment:
hrm.. just curious why mysql needs extra care re: the scc and mongo does not.  Both mysql and mongo use the same setup for scc.
https://github.com/konveyor/mig-demo-apps/blob/master/apps/todolist-mariadb-go/mysql-persistent-csi-template.yaml#L31
https://github.com/konveyor/mig-demo-apps/blob/master/apps/todolist-mongo-go/mongo-persistent-csi-template.yaml#L17

Reply:
extra care was there so, I did not remove for mySql. If require, I can add that for mongo.

Reply:
I think whoever added the test case was just checking if everything was created. Which is not a bad idea.

Reply:
please just make it consistent for both apps. :)

Reply:
@weshayutin  looks consistent!